policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/living_tax_credit/max_amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Northwest Territories allows for this maximum amount under the cost of living tax credit.
+values:
+  2022-12-31: 942 
+metadata:
+  unit: currency-CAD
+  label: Northwest Territories cost of living tax credit max amount
+  reference:
+  - title: Government of Canada - Credit
+    href: https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/northwest-territories/5012-tc.html 
+          https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/northwest-territories/5012-pc/information-residents-northwest-territories.html 
```

Comment:
need to change reference to the same format as the first one:
- title:
   href:

Reply:
both yaml files